bundles/binding/org.openhab.binding.isy/README.md

```diff
@@ -0,0 +1,90 @@
+## Description
+
+This is a openHAB binding for the Universal Devices ISY 994i router and PLM. Universal Devices offers a Java SDK for
+external programs to communicate with the router. Alternatives include an extensive REST API. See details at
+https://www.universal-devices.com/isy-developers/.
+
+## Instructions
+
+To build and install the isy binding, you need to first download the Java SDK that corresponds with the firmware version
+of your ISY 994i.
```

Comment:
More developer notes, moved and updated it to make more sense. If there is a later firmware you want to build against this needs to be done and having these notes may help other developers.